### DISCUSSION TITLE
Amazon IAP

### DISCUSSION THREAD

**Discussion by Kabilan34** [hyochan/react-native-iap#1537] [Q&A]
the existing Amazon IAP does not support purchase of subscriptions as expected.

the Receipt.class does not properly handle the response from Amazon where term and termSku values are returned.
This could block people from doing receipt validation for the purchases.

Is there any solution for this or when can a fix can be expected?

**Comment by hyochan** [comment]
Would like to handle this thread in https://github.com/hyochan/expo-iap/discussions/239
